cddl/reference-endorsed-triple-record.cddl

```diff
@@ -0,0 +1,30 @@
+;
+; Triples map extension
+;
+ $$triples-map-extension //= (
+  &(reference-endorsed-triples: 7) => [ + reference-endorsed-triple-record]
+)
+;
+; Reference Endorsed Triple Record
+;   Subject: Target Environment ($TE)
+;   Object: Reference Values 'refvals' ($RV) + Endorsed Values 'endvals' ($EV)
+;   Predicate: When $TE matches evidence, the $RV claims are accepted 
+;                   and subsequently the $EV claims are also accepted
+;
+reference-endorsed-triple-record = [
```

Comment:
Propose to add support of time series in this new reference-endorsed-triple-record:

reference-endorsed-triple-record = [
environment-map ; target environment
[ * reference-endorsed-map ] ; multi-valued list, which can be an empty list
? [ + reference-endorsed-map ] ; If this second list is present, it contains a time series of reference-endorsed-maps
]

If each entry in a time series should be a multi-valued list, then the definition will be:

reference-endorsed-triple-record = [
environment-map ; target environment
[ * reference-endorsed-map ] ; multi-valued list, which can be an empty list
? [ + [ + reference-endorsed-map ] ] ; If this second list is present, it contains a time series of reference-endorsed-map lists.
]